.github/workflows/blui-ci.yml

```diff
@@ -32,10 +32,8 @@ jobs:
                   node-version: ${{ matrix.node-version }}
                   cache: 'yarn'
             - run: yarn
-            # - run: yarn docs-prettier
-            #   working-directory: docs
-        #   - run: yarn docs-lint
-        #     working-directory: docs
+            - run: yarn prettier:check
+            - run: yarn docs-lint
```

Comment:
shouldn't this be `yarn lint`

Reply:
`docs-lint` and `lint` are the same thing. I can change it

Reply:
`docs-lint `runs for only docs project 
`lint` runs for all projects